salt/config/__init__.py

```diff
@@ -43,8 +43,9 @@
 
 try:
     import psutil
+    psutil.virtual_memory
```

Comment:
I added a new version of this cache. See:  https://github.com/saltstack/salt/pull/35726/commits/3c6d9dfb4e7e27d5218b51562f82675de47ccf8e